jenkins-design-language/src/js/components/material-ui/svg-icons/action/accessibility.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import pure from 'recompose/pure';
```

Comment:
But yes, we definitely don't want to do anything to hurt upgrading React - v16 (IIRC) with the fiber engine allows components to return multiple children from `render()`, which will allow us to clean up a bunch of our HOCs that cause layout issues due to wrapping divs that are only there to satisfy the "single div" rule.